packages/commonwealth/client/scripts/models/IChainAdapter.ts

```diff
@@ -96,12 +96,10 @@ abstract class IChainAdapter<C extends Coin, A extends Account> {
   public deinitServer() {
     this._serverLoaded = false;
     this.app.threads.deinit();
-    this.app.comments.deinit();
-    this.app.comments.deinitReactionsStore();
+    // this.app.threads.deinit(); // TODO: do we want to clear react query state here?
```

Comment:
Question @jnaviask @masvelio @CowMuon

Reply:
I also incline toward the faster load since we have been keying by `chainId` during the react query migration, so I think we should not clear react query state when we switch b/w communities (logout seems like an exception).

Removed the commented code.